kernal/cbm/irq.s

```diff
@@ -20,9 +20,19 @@
 .import led_update
 .export panic
 
+.include "banks.inc"
+
+rom_bank = 1
+
+
 ; puls - checks for real irq's or breaks
 ;
 puls	pha
+	lda rom_bank
+	beq :+
```

Comment:
Anonymous label as opposed to `.ifp02`? (Does anywhere else even use `.ifp02`?)

Reply:
This is for the legacy C64 compat for the KERNAL. I believe that everything has diverged so much with no updates to the C64-specific files that the C64 version will no longer build.  Even mist himself has said so some time ago.  We might want to start cleaning that up after the next iteration.